### PR TITLE
Add Post pages.

### PR DESCRIPTION
**Pull request by ziggabyte**
This PR adds individual pages for Posts, that you get to by clicking on a Post-preview in the PostFeed. They are located at /posts/(slug). The slugs are generated automatically from the title, but are changable.

### CODE REVIEW

File: cms/src/api/post/content-types/post/lifecycles.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+};
+
+module.exports = {
+  async beforeCreate(newPost) {
+    if (newPost.params.data.slug) {
+      newPost.params.data.slug = slugify(newPost.params.data.slug, {
+        lower: true,
+      });
+    } else {
+      newPost.params.data.slug = slugify(newPost.params.data.title, {
```
Comment (Bagera) [inline cms/src/api/post/content-types/post/lifecycles.js:21]: Looks good. Just a question: how does slugify handle åäö by default? I think we should just remove the dots.

File: frontend/src/pages/posts/[postSlug].tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+          sx={{
+            color: 'primary.main',
+          }}
+        >
+          <PageHeader
+            title={post.attributes.title}
+            image={post.attributes.image}
+          />
+          <Typography>
+            {dayjs(post.attributes.publishedAt).format('DD/MM/YYYY')}
```
Comment (Bagera) [inline frontend/src/pages/posts/[postSlug].tsx:54]: This should use the function you made yesterday

**Review by Bagera — CHANGES_REQUESTED** [review]
Small changes and questions

File: cms/src/api/post/content-types/post/lifecycles.js (hunk 1 above)
Comment (ziggabyte) [reply cms/src/api/post/content-types/post/lifecycles.js:21]: when i tested it, i turns them into o and a!

File: frontend/src/pages/posts/[postSlug].tsx (hunk 2 above)
Comment (ziggabyte) [reply frontend/src/pages/posts/[postSlug].tsx:54]: ofcourse! :D

**Review by Bagera — APPROVED** [review]
Looking good

**Pull request merged**